packages/react/src/constants/CardPropTypes.js

```diff
@@ -688,4 +688,8 @@ export const CardPropTypes = {
   locale: PropTypes.string,
   /** a way to pass down dashboard grid resize handles, only used by other card types */
   resizeHandles: PropTypes.array,
+  /** Optional callback function that is passed an onChange function and the original cardConfig function.
+   * This allows additional information to be passed to be used in the Card Editor for this type.
+   * You need to return an array of child objects with a header: {title, tooltip: {tooltipText: PropTypes.string}} and content element to render * */
+  renderEditContent: PropTypes.func,
```

Comment:
Suggestion : 

`renderEditContent` can be renamed into `renderExtraContent`

Reply:
I think we have to leave it with Edit since it was already delivered that way I'm just documenting that property